app/assets/javascripts/covid19/src/data/LigandsApiRepository.ts

```diff
@@ -0,0 +1,89 @@
+import _ from "lodash";
+import { LigandImageData } from "../domain/entities/LigandImageData";
+import { LigandsRepository } from "../domain/repositories/LigandsRepository";
+import { routes } from "../routes";
+import { Future, FutureData } from "./utils/future";
+import { getJSON } from "./utils/request-utils";
+
+export class LigandsApiRepository implements LigandsRepository {
+    ligandsData: any[] = [];
```

Comment:
My bad, I put any everywhere because we were expecting the API to change. By the moment I'm writing this comment, it seems the API has already changed the endpoint, but the IDR scheme of every ligand vanished. I'm waiting for the final scheme, and then I will use the purify-ts codec as we discussed on slack.